radicle-httpd/src/api/v1/projects.rs

```diff
@@ -45,9 +45,11 @@ pub fn router(ctx: Context) -> Router {
                 )),
             ),
         )
+        .route("/projects/:project/tree/:sha/", get(tree_handler_root))
```

Comment:
In 0.5, `/projects/:project/tree/:sha/*path` would also match when hitting `/projects/rad:.../tree/01.../` but not in 0.6. You either have to add a separate handler for the root version or use `route_with_tsr`